website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -291,6 +294,7 @@ spec:
 ```
 
 </CodeBlockConfig>
+-> **NOTE**: L7 Intentions are not currently supported for transparent proxy Destinations.
```

Comment:
What are L7 Intentions? There's not a good page to link to unfortunately but we could at least link to https://consul-17na2ubrq-hashicorp.vercel.app/docs/connect/config-entries/service-intentions#permissions